engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/fenced-pages.service.js

```diff
@@ -15,6 +15,7 @@ angular.module('Bastion.organizations').service('FencedPages',
             'subscriptions',
             'gpg-keys',
             'sync-plans',
+            'sync-plan',
```

Comment:
I didn't recall seeing a similar update for other entities which is why it stuck out to me (e.g. gpg keys or products)